pkg/hostagent/requirements.go

```diff
@@ -132,6 +132,19 @@ Also see "/var/log/cloud-init-output.log" in the guest.
 A possible workaround is to run "lima-guestagent install-systemd" in the guest.
 `,
 	})
+	if *a.y.SSH.ForwardAgent {
```

Comment:
I'm not sure if this should be an `essentialRequirements` or an `optionalRequirements `.

Reply:
Ah, good point. I think this should go into `optionalRequirements` instead.

The essential requirements are things that are needed for Lima itself to work properly (e.g. `sshfs` may need to be installed for the mounts to work). Everything that is only needed for user operations should be in "optional".

Reply:
Moved to `optionalRequirements`.